app/server/server_test.go

```diff
@@ -246,16 +250,19 @@ func TestServer_getParams(t *testing.T) {
 
 func prepTestServer() (ts *httptest.Server, teardown func()) {
 	eng := store.NewInMemory(time.Second)
-	srv := Server{
-		Messager: messager.New(eng, messager.Crypt{Key: "123456789012345678901234567"}, messager.Params{
+
+	srv, _ := New(
```

Comment:
nit: we better not ignore the error here, but check it with require. To make this possible, adding the t param to this prepTestServer may be necessary.

Reply:
done